tests/queries/0_stateless/01529_bad_memory_tracking.sh

```diff
@@ -8,6 +8,6 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 . "$CURDIR"/../shell_config.sh
 
 for _ in {1..10}; do
-    ${CLICKHOUSE_CLIENT} --max_memory_usage '10G' --query "SELECT i FROM generateRandom('i Array(Int8)', 1, 1, 1048577) LIMIT 65536" |& grep -v -e 'Received exception from server' -e 'Code: 241' -e '(query: '
+    ${CLICKHOUSE_CLIENT} --max_block_size=65505 --max_memory_usage '10G' --query "SELECT i FROM generateRandom('i Array(Int8)', 1, 1, 1048577) LIMIT 65536" |& grep -v -e 'Received exception from server' -e 'Code: 241' -e '(query: '
```

Comment:
btw, from which range do we peek a random value for `max_block_size`?

Reply:
https://github.com/ClickHouse/ClickHouse/blob/8de655bf1fdf2ddaaec81cc645f7513d392f64f3/tests/clickhouse-test#L379